regex-assembly/920120.ra

```diff
@@ -24,4 +24,4 @@
 &apos;
 &nbsp;
 &oslash;
-[^'\";=\x5c]
+[^\";=\x5c\x3c\x3e\{\}]
```

Comment:
No backslashes necessary. Most meta characters are not ambiguous inside of a character class.
```suggestion
[^";=\x5c<>{}]
```